pages/common/act.md

```diff
@@ -0,0 +1,28 @@
+# act
+
+> Execute GitHub Actions locally using Docker.
+> More information: <https://github.com/nektos/act>.
+
+- List the available actions:
+
+`act -l`
+
+- Run the default event:
+
+`act`
+
+- Run a specific event:
+
+`act {{event_type}}`
+
+- Run a specific action:
+
+`act -a {{action_id}}`
+
+- Do not actually run the actions (i.e. a dry run):
+
+`act -n`
+
+- Enable verbose logging:
```

Comment:
"Enable" might not be very accurate here as it might mean that it's like a switch that you have to disable again. I would just say "Show verbose logs"